src/shogun/preprocessor/FisherLDA.cpp

```diff
@@ -68,8 +68,6 @@ void CFisherLDA::initialize_parameters()
 	m_num_dim=0;
 	m_gamma = 0;
 	m_bdc_svd = true;
-	SG_ADD(
-	    &m_method, "FLDA_method", "method for performing FLDA");
```

Comment:
`m_method` is added again a couple of lines later. I thought it's the cause of a problem, but it's fixed now, and I don't remember whether that was the problem or not :"D